gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -32,7 +32,8 @@ def parse_systemd
     end
 
     def files(unit_extension)
-      (SYSTEM_DIRS + USER_DIRS).flat_map do |dir|
+      dirs = my_dirs
+      dirs.flat_map do |dir|
```

Comment:
Yes, this should be in it's own PR. You should probably open an issue describing the problem as well.